.github/workflows/build_and_test.yml

```diff
@@ -2,10 +2,10 @@ name: Build and Tests
 
 on: [push, pull_request]
 
-jobs: 
+jobs:
   build-and-test:
     name: Build and Test
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
We want the tests to match production as close as possible, which is currently 18.04.  I will update to 22.04 on the staging to see if it causes us any issues.  Can you pin this to 22.04?

Reply:
Note that flake8 action is having the same issue.